platform/jewel/ui/src/main/kotlin/org/jetbrains/jewel/ui/component/SimpleListItem.kt

```diff
@@ -130,7 +130,7 @@ public fun SimpleListItem(
         verticalAlignment = Alignment.CenterVertically,
         horizontalArrangement = Arrangement.spacedBy(style.metrics.iconTextGap),
     ) {
-        if (icon != null) {
+        if (icon != null && iconContentDescription != null) {
```

Comment:
This is wrong. The icon should be visible even if it does not have a content description. What is the logic behind this?